packages/dcos-history/extra/history/statebuffer.py

```diff
@@ -7,10 +7,26 @@
 
 import requests
 
-FETCH_URL = "http://leader.mesos:5050/state-summary"
+logging.getLogger('requests.packages.urllib3').setLevel(logging.WARN)
+
 FETCH_PERIOD = 2
 FILE_EXT = '.state-summary.json'
 
+FETCH_URL = "http://leader.mesos:5050/state-summary"
+if 'STATE_SUMMARY_URI' in os.environ:
```

Comment:
This doesn't make a lot of sense as a separate / different STATE_SUMMARY_URI. Can we make it have one name throught? Either always FETCH_URL or always STATE_SUMMARY_URL?